modules/gridmap/grid_map.cpp

```diff
@@ -547,6 +547,123 @@ GridMap::OctantKey GridMap::get_octant_key_from_cell_coords(const Vector3i &p_ce
 	return ok;
 }
 
+Vector3i GridMap::get_octant_coords_from_cell_coords(const Vector3i &p_cell_coords) const {
+	return Vector3i(
```

Comment:
Maybe cache `float sizeMinusOne = octant_size - 1;` and `float inverseSize = 1.f / octant_size;`  may be shorter and more readable, and debug builds will not do 3 divs

Reply:
One point in the review further up was to remove the use of float. This now is a straight copy of the TileMap cell raster code. If it makes sense to change it again to a cached version for performance should preferably be its own PR with performance tests that change both.